### PR TITLE
Basic github actions workflow for testing

### PR DESCRIPTION
**Pull request by wlach**
Not sure if you have a preference for CI systems, but I think github
actions works pretty well-- and will help us make sure that commits /
pull requests don't break things.

### CODE REVIEW

**Comment by wlach** [comment]
You may need to enable github actions on this repository for the above to do the right thing.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        node-version: [14.x, 15.x, 16.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
```
Comment (dmaevsky) [inline .github/workflows/test.yml:27]: There's no `npm ci` script, will this work?

Comment (wlach) [reply .github/workflows/test.yml:27]: npm ci is a built-in: https://docs.npmjs.com/cli/v7/commands/npm-ci

Comment (dmaevsky) [reply .github/workflows/test.yml:27]: got it, TIL :)

**Comment by dmaevsky** [comment]
Hm, it doesn't seem to work with yarn.lock though.
I am ok to switch to npm. No preference here

**Comment by wlach** [comment]
> Hm, it doesn't seem to work with yarn.lock though. I am ok to switch to npm. No preference here

No real preference here either. :) If the rest of your toolchain is yarn, maybe best to just be consistent. I'll submit a followup later today.

**Comment by dmaevsky** [comment]
Just made the switch. Recently they are pretty interchangeable, and I found
some weird behavior in yarn which NPM did not exhibit the other day, so
even if I'm used to yarn, I'm leaning towards going back the NPM way.
Anyway, tests pass now.
Great success! ;)
Thanks a lot for your help!

**Pull request merged**